### ISSUE TITLE
New issue on windows insider build 15033.

### ISSUE THREAD

**Issue by lovettchris** [Microsoft/BashOnWindows#1710]
sudo apt-get upgrade 

0% [Working]http: ../sysdeps/posix/getaddrinfo.c:2583: getaddrinfo: Assertion `(__extension__ ({ const struct in6_addr *__a = (const struct in6_addr *) (sin6->sin6_addr.__in6_u.__u6_addr32); __a->__in6_u.__u6_addr32[0] == 0 && __a->__in6_u.__u6_addr32[1] == 0 && __a->__in6_u.__u6_addr32[2] == __bswap_32 (0xffff); }))' failed.
E: Method http has died unexpectedly!
E: Sub-process http received signal 6.

**Comment by lovettchris** [comment]
Seems BashOnWindows on insider builds starting with 15033 is toast.  I can't do any sudo apt-get update, or upgrade.  I just got updated to 15040, same issue.

0% [Working]http: ../sysdeps/posix/getaddrinfo.c:2583: getaddrinfo: Assertion `(__extension__ ({ const struct in6_addr *__a = (const struct in6_addr *) (sin6->sin6_addr.__in6_u.__u6_addr32); __a->__in6_u.__u6_addr32[0] == 0 && __a->__in6_u.__u6_addr32[1] == 0 && __a->__in6_u.__u6_addr32[2] == __bswap_32 (0xffff); }))' failed.

**Comment by lovettchris** [comment]
What should I do to recover?  I will try a full lxrun /uninstall, /install and see if that helps...

**Comment by lovettchris** [comment]
Nope, full reset didn't help.

$ sudo apt-get update
0% [Working]http: ../sysdeps/posix/getaddrinfo.c:2603: getaddrinfo: Assertion `IN6_IS_ADDR_V4MAPPED (sin6->sin6_addr.s6_addr32)' failed.
Reading package lists... Done
E: Method http has died unexpectedly!
E: Sub-process http received signal 6.

**Comment by fpqc** [comment]
I don't even see 15033 available on the updater.

**Comment by lovettchris** [comment]
it's a Windows Insider build, see "Windows Insider Program" under your Windows Settings.  I'm now on 15040 and I have the same problem.  I'm completely blocked by this.

**Comment by fpqc** [comment]
@lovettchris yeah, those are both microsoft-internal builds.  15043 is the current public build. It also appears to be release preview 1, (no more test build notice on the bottom right).

**Comment by lovettchris** [comment]
[update_trace.zip](https://github.com/Microsoft/BashOnWindows/files/801592/update_trace.zip)

strace is attached.

**Comment by aseering** [comment]
@lovettchris -- I don't have this problem either.  You are running an older Insider build; try updating?

**Comment by fpqc** [comment]
@aseering, it might be a regression that was introduced in internal builds after 15031, then fixed before the latest public build, 15042. If you look at his tag, @lovettchris is a Microsoft employee and they get more builds flighted to them than we in the general public do.  But yes, it does appear to be resolved now.

**Comment by aseering** [comment]
@fpqc , "regression":  yeah, makes sense.  I thought I had seen one or two tickets from Microsoft employees reporting the same thing, but I can't seem to find them.

**Comment by lovettchris** [comment]
Ok, I am no longer blocked, I went back to non-insider build (anniversary edition).
